tests/integration_tests/README.md

```diff
@@ -5,6 +5,16 @@ These tests will delete and modify data permanently. Expect the localhost instan
 
 See: [localhost configuration](files/configuration/localhost_config.json)
 
+## Running locally
+
+To set up a local test environment, use the test environment and docker-compose files in the root of the repo:
+
+```
+docker compose --env-file test.env -f docker-compose-test.yml up
```

Comment:
Because I dislike committing `.env` files to source control. IMO it makes it easier for a dev to write their own environment file if necessary, and they can use their own `.env` that won't get picked up by git if they choose to. These env vars are configured separately in the GH workflow.

Reply:
But for testing specifically, shouldn't we all be sharing a .env?

Reply:
Fair point...probably, I should just replace the secrets with hardcoded values in the docker compose file since none of those values are actually secret credentials for anything in particular.

Reply:
I caved and just renamed the file to `.env` 🙂